pkg/policy/validate.go

```diff
@@ -132,90 +133,103 @@ func Validate(policy *kyverno.ClusterPolicy, client *dclient.Client, mock bool,
 	for i, rule := range policy.Spec.Rules {
 		//check for forward slash
 		if err := validateJSONPatchPathForForwardSlash(rule.Mutation.PatchesJSON6902); err != nil {
-			return fmt.Errorf("path must begin with a forward slash: spec.rules[%d]: %s", i, err)
+			return nil, fmt.Errorf("path must begin with a forward slash: spec.rules[%d]: %s", i, err)
 		}
 
 		if jsonPatchOnPod(rule) {
 			log.Log.V(1).Info("pods managed by workload controllers cannot be mutated using policies, use the auto-gen feature or write policies that match pod controllers")
+			return &v1beta1.AdmissionResponse{
+				Allowed:  true,
+				Warnings: []string{"pods managed by workload controllers cannot be mutated using policies, use the auto-gen feature or write policies that match pod controllers"},
```

Comment:
Run-on sentence in English. Suggested:

"Pods managed by workload controllers cannot be mutated using policies. Use the autogen feature or write policies that match Pod controllers."

Reply:
@chipzoller I copied it from the above [line](https://github.com/kyverno/kyverno/blob/6b5bcfcc42adb409208f63895c5bbc131153bd6b/pkg/policy/validate.go#L139) to keep the warning consistent with the log message. Would you like me to update the log message as well or leave it as is?

Reply:
Yes, the English grammar needs correcting wherever it occurs.

Reply:
Updated the existing log message as well to use the suggested message.